spec/integrations/shared/base_render_examples.rb

```diff
@@ -10,6 +10,20 @@
     it('returns json with specified fields') { should eq(result) }
   end
 
+  context 'Given blueprint has ::field with all data types' do
```

Comment:
I wanted to demonstrate the failing boolean test first, but I'd be happy to extend this spec to ensure we have coverage of array and hash field data types as well